be-golang-plain/files/docker/Dockerfile

```diff
@@ -1,33 +1,7 @@
-FROM alpine
-
-# Uncomment this section if you need to use glibc in your container, f.e. when you enable CGO_ENABLED=1
```

Comment:
Did you already check if glibc is in the base image as otherwise this would cause  https://github.com/opendevstack/ods-quickstarters/issues/277 ?

Reply:
Glibc package is not avaialble in a more up to date version in the repositories to install it for what I could see.
Building from source is also complicated (not successful in my tests) and resource-hungry process.

For now this glibc version should be enough for use cases